### ISSUE TITLE
Some characters in quotes do not get read back correctly from the database

### ISSUE THREAD

**Issue by bitfolk** [bitfolk/enoch#12]
Quotes that are definitely added correctly:

```
<grifferz> !addquote <find> http://a.vu/aifl - BBC News - Clinton
           Groom death in Barton-le-Clay 'not suspicious' <ʇɹǝqoɹ>
           we were in the same class in year 9 <ʇɹǝqoɹ> i think he
           once gave me a power wedgie <Dominic> this is what
           happens when you bully Robert, he plays the long game
-Enoch:#bitfolk- Added quote 6039 with an initial rating of 7.
```

Come back with mangled characters:

```
<grifferz> !quote 6039
-Enoch:#bitfolk- Quote[6039 / 7.0 / grifferz @ 31-Oct-12]: <find>
                 http://a.vu/aifl - BBC News - Clinton Groom death
                 in Barton-le-Clay 'not suspicious'
                 <Ê<U+0087>É¹Ç<U+009D>qoÉ¹> we were in the same
                 class in year 9 <Ê<U+0087>É¹Ç<U+009D>qoÉ¹> i think
                 he once gave me a power wedgie <Dominic> this is
                 what happens when you bully Robert, he plays the
                 long game
```

**Comment by bitfolk** [comment]
It looks like the `encode_utf8()` call was a mistake. Seems to be encoding it twice.

This may have been fixed by SHA: b671bb9af3987e1cb43e8b4981b309ac4d3c997c. If not, at least it will give more debug info.

**Comment by grifferz** [comment]
Appears fixed.
